docs/aws-lambda-extension.asciidoc

```diff
@@ -21,137 +21,68 @@ By using a custom-built AWS Lambda Extension, Elastic APM Agents can send data t
 
-The name of your Lambda function.  The installer will use this to configure the correct Lambda function.  This must be the name of a function that already exists.
+* The AWS region your Lambda function runs in. The APM Lambda Extension layer needs to be in the same region as your Lambda function.
+* The architecture (_x86_64_ or _arm64_) used for your Lambda function.
+* The version of the APM Lambda Extension you would like to use.
 
-[discrete]
-[[aws-lambda-lambda_env]]
-=== `lambda_env`
+You'll then need to configure your function to use that layer. To add a layer
 
-The installer will use the key/value pairs in this section of the configuration file to add environment variables to your Lambda function.  The provided variables are those required to make the extension work correctly.
+1. Navigate to your function in the AWS Console
+2. Scroll to the Layers section and click the _Add a layer_ button image:images/config-layer.png[image of layer configuration section in AWS Console]
+3. Choose the _Specify an ARN_ radio button
+4. Enter the Version ARN of the APM Lambda Extension layer in the _Specify an ARN_ text input
+5. Click the _Add_ button
 
 [discrete]
-[[aws-lambda-log_level]]
-==== `ELASTIC_APM_LOG_LEVEL`
+[[aws-lambda-env-vars]]
+==== Configure Environment Variables
 
-The log level for the APM Agent.  Consult your https://www.elastic.co/guide/en/apm/agent/index.html[APM Agent's documentation] for more information.
+Finally, once the layer's in place you'll need to configure a few environment variables. To configure variables
 
-[discrete]
-[[aws-lambda-secret_token]]
-==== `ELASTIC_APM_SECRET_TOKEN`
+1. Navigate to your function in the AWS Console
+2. Click on the _Configuration_ tab
+3. Click on _Environment variables_
+4. Add the necessary variables.
 
-The APM secret token.  The extension will use this when communicating with APM Server.
+The following (required and optional) environment variables are relevant for instrumenting a Lambda function:
 
 [discrete]
-[[aws-lambda-api_key]]
-==== `ELASTIC_APM_API_KEY`
+[[aws-lambda-lambda_apm_server]]
+===== `ELASTIC_APM_LAMBDA_APM_SERVER` (required)
 
-An alternative authentication method to the secret token.  The extension will use this when communicating with APM Server.
+The `ELASTIC_APM_LAMBDA_APM_SERVER` controls where the Lambda extension will ship data.  This should be the URL of the final APM Server destination for your telemetry.
 
 [discrete]
-[[aws-lambda-server_url]]
-==== `ELASTIC_APM_SERVER_URL`
+[[aws-lambda-apm_secret_token]]
+===== `ELASTIC_APM_SECRET_TOKEN` or `ELASTIC_APM_API_KEY` (required)
 
-Your APM Server URL.  This is the final destination for your data.
+Either `ELASTIC_APM_API_KEY` or `ELASTIC_APM_SECRET_TOKEN` needs to be set.  This controls the authentication method that the extension uses when sending data to the URL configured via `ELASTIC_APM_LAMBDA_APM_SERVER`.
 
-[discrete]
-[[aws-lambda-service_name]]
-==== `ELASTIC_APM_SERVICE_NAME`
+[descrete]
```

Comment:
```suggestion
[discrete]
```

Reply:
I'm not sure if this is why the docs build warning: https://elasticsearch-ci.elastic.co/job/elastic+apm-aws-lambda+pull-request+build-docs/77/console

```
02:14:23 INFO:build_docs:asciidoctor: WARNING: ../../apm-aws-lambda/docs/aws-lambda-extension.asciidoc: line 72: section title out of sequence: expected level 3, got level 6
```

/shrug

Reply:
Yup! This should fix the doc build.